searx/utils.py

```diff
@@ -673,6 +673,22 @@ def get_embeded_stream_url(url):
             video_id = path_parts[2]
             iframe_src = 'https://www.dailymotion.com/embed/video/' + video_id
 
+    # Bilibili
+    elif parsed_url.netloc in ['www.bilibili.com', 'bilibili.com'] and parsed_url.path.startswith('/video/'):
+        path_parts = parsed_url.path.rstrip('/').split('/')
```

Comment:
`rstrip('/')` is completely unnecessary here, `split` starts from the left anyways and removes all occurrences of `/`, so what's at the end doesn't matter